drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -139,6 +139,12 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
         default=False,
     )
 
+    max_grade = Float(
```

Comment:
Can we rename this to just `grade`? `max_grade` makes it sound like a setting for maximum possible grade (what we currently use `weight` for). In fact I think CAPA problems use `max_grade` field to mean "maximum possible grade", not "maximum achieved grade".